src/doc/src/reference/external-tools.md

```diff
@@ -265,6 +265,10 @@ cargo `(?<command>[^ ]+)` into an invocation of an external tool
 `cargo-${command}`. The external tool must be present in one of the user's
 `$PATH` directories.
 
+> Note: By default, Cargo invokes custom subcommand in `$CARGO_HOME/bin`
+> instead of `$PATH`. If you want to change this behavior, you can put
+> `$CARGO_HOME/bin` in `$PATH` to control the precedence.
```

Comment:
"instead of" doesn't convey the precedence. How about this?

```suggestion
> **Note**: Cargo defaults to prioritizing external tools in `$CARGO_HOME/bin`
> over `$PATH`. Users can override this precedence by adding `$CARGO_HOME/bin`
> to `$PATH`.
```

Reply:
I've used what you suggested and it looks pretty good without any grammar problems